misc/services/mobile-webui/mobile-webui-frontend/src/reducers/headers.js

```diff
@@ -48,11 +50,18 @@ export default function reducer(state = initialState, action) {
         location: { pathname },
       } = payload;
 
-      const newEntries = removeEntries({
-        entriesArray: state.entries,
-        startLocation: pathname,
-        inclusive: false,
-      });
+      let newEntries = null;
+
+      // clear header on main urls
+      if (pathname === '/' || launchersUrlRegExp.test(pathname)) {
```

Comment:
cant we get rid of hardcoded lauchersUrlRegex?

Reply:
Not really, no. Unless we want to have the header persist:

![Screenshot 2021-11-03 at 18 42 32](https://user-images.githubusercontent.com/513460/140163479-3038755b-cf76-486f-aa30-3f894554f869.png)

Maybe it worked before when we had the launchers screen at `/`, but now it's `/launchers/<appId>`, and there's also the app selection screen. I tried changing the initialState's entry to what we use right now but it's not working. So I expect there's more digging needed to get this to work the same way it did before. That's why I think this approach is easier.

Reply:
understand